Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -47,12 +47,56 @@
     return s_dwriteFactory;
 }
 
+template <typename TElement>
+bool __CloneArray(_In_reads_opt_(count) TElement const* source,
```

Comment:
stl::copy only supports iterator based copy if I am not wrong?

---
In reply to: [78468942](https://github.com/Microsoft/WinObjC/pull/999#discussion_r78468942) [](ancestors = 78468942)